string type
string recruiter_id
string sender_id
float32 x
float32 y
float32 z
float32 distance
